sktime/forecasting/theta.py

```diff
@@ -262,3 +268,160 @@ def _zscore(level: float, two_tailed: bool = True) -> float:
     if two_tailed:
         alpha /= 2
     return -norm.ppf(alpha)
+
+
+class ThetaModularForecaster(BaseForecaster):
+    """Modular theta method for forecasting.
+
+    Modularized implementation of Theta method as defined in [1]_ (TODO: add the
+    auto-theta method as described in [2]_).
+
+    Overview: Input :term:`univariate series <Univariate time series>` of length
+    "n" and decompose with :class:`ThetaLinesTransformer
+    <sktime.transformations.series.theta>` by modifying the local curvature of
+    the time series using Theta-coefficient values - `theta_values` parameter.
+    Thansformation gives a pd.DataFrame of shape `len(input series) * len(theta)`.
+
+    The resulting transformed series (Theta-lines) are extrapolated separately.
+    The forecasts are then aggregated into one prediction - aunivariate series,
+    of `len(fh)`.
+
+    Parameters
+    ----------
+    forecasters: list of tuples (str, estimator, int or pd.index), default=None
```

Comment:
... you could explain better that `forecasters` and `theta_values` are "paired" with each other.